controllers/TaskController.php

```diff
@@ -218,4 +219,19 @@ public function actionDrop($taskId)
 
         return $this->asJson(['success' => false]);
     }
+
+    public function actionChooseWorkerAccount($id, $accountId)
```

Comment:
what is this one doing?

Reply:
It's a simple action that handles an AJAX CALL to select a target account  ( payment account ) for a task.
Cf. picture below (Choose Payment Account Button)  :
![FireShot Capture 177 - Test Space - Coinsence_ - http___coinsence_index php](https://user-images.githubusercontent.com/13172404/54093512-dd653c00-4398-11e9-9577-015d71c7a122.png)